lib/windows/seekdialog.py

```diff
@@ -664,7 +664,10 @@ def doSeek(self, offset=None, settings_changed=False):
             self.handler.seek(self.selectedOffset if offset is None else offset, settings_changed=settings_changed)
 
             if state_before_seek == self.player.STATE_PAUSED:
-                self.player.control("pause")
+                if not settings_changed:
+                    self.player.control("pause")
+                else:
+                    self.player.pauseAfterPlaybackStarted = True
```

Comment:
I think probably this should all be removed from here and all handled in player.py. Whenever we're paused when we start a seek we should be paused after and the `self.player.control("pause")` I had here only works if you're not transcoding. With the  PR as it is, if you're pause before a seek and transcoding it will still seek and play.

Reply:
(I realize this was code was wrong before you touched it :smile:)

Reply:
> With the PR as it is, if you're pause before a seek and transcoding it will still seek and play.

Weird. I tested exactly this with my newest changes and it worked flawlessly.

Reply:
But it makes sense to put everything into player.py, yeah.